jerry-core/jerryscript-port.h

```diff
@@ -140,16 +140,31 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
-#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
-
-#define JERRY_PORT_ENABLE_JOBQUEUE
```

Comment:
@martijnthe Do I understand correctly that you are against this change? (Sorry, non-native speaker.) If so, my justification(s):
- `jerryscript-port.h` is a public header. Assume that `jerry-core` is distributed in binary form, then all you have is the `libjerry-core.a` binary archive and this header. Then, when you include that header, you have no information on how the lib was configured and built, whether it actually contains ES2015 implementations or not. So, adding config conditionals (`#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN`) to a public header is not a good idea. But if you don't have that conditional, then `JERRY_PORT_ENABLE_JOBQUEUE` is always defined, which then adds no information.
- Even if a port implements the job queue but nothing references the implementing functions, then the linker will not link the implementing object into the project.

(Or do I get soething wrong?)